packages/shared-base/src/utils/compose.ts

```diff
@@ -0,0 +1,5 @@
+import React from 'react'
+
+export function compose(init: React.ReactNode, ...f: ((children: React.ReactNode) => JSX.Element)[]) {
```

Comment:
This MUST NOT be in the shared-base package because it imports React. Please move it to `shared` package.